src/datatable/expr/reduce.py

```diff
@@ -119,7 +119,12 @@ def sum(iterable, start=0):
 
 # noinspection PyShadowingBuiltins
 def min(*args, **kwds):
-    if len(args) == 1 and (not isinstance(args[0], dict)) and (isinstance(args[0], core.FExpr) or isinstance(args[0][0], core.FExpr)):
+    if (len(args) == 1 
+        and (not isinstance(args[0], dict)) 
+        and (isinstance(args[0], core.FExpr)
+            or ((hasattr(args[0], "__getitem__") and isinstance(args[0][0], core.FExpr)))
+           )
+        ):
```

Comment:
So what was the reason to update this part? Readability?

Reply:
restored to original form

Reply:
I am not actually against improving readability, just asking.

Reply:
No issues with the initial format ... I only split it to understand the if statement and forgot to restore it to the initial form